lib/gcloud/vision/analysis.rb

```diff
@@ -199,142 +255,192 @@ def labels
+      #   image = vision.image "./acceptance/data/text.png"
+      #
+      #   analysis = vision.annotate image, text: true
       #   analysis.text? #=> true
       #
       def text?
         !text.nil?
       end
 
-      # The Analysis::SafeSearch results containing the results of safe_search
-      # detection.
+      ##
+      # The results of safe_search detection.
+      #
+      # @return [SafeSearch]
       #
       # @example
       #   require "gcloud"
       #
       #   gcloud = Gcloud.new
       #   vision = gcloud.vision
-      #   analysis = vision.detect image, safe_search: true
+      #   image = vision.image "./acceptance/data/face.jpg"
+      #
+      #   analysis = vision.annotate image, safe_search: true
       #   safe_search = analysis.safe_search
       #
       def safe_search
         return nil unless @gapi["safeSearchAnnotation"]
         @safe_search ||= SafeSearch.from_gapi(@gapi["safeSearchAnnotation"])
       end
 
-      # Whether there is a Analysis::SafeSearch result for safe_search
+      ##
+      # Whether there is a result for safe_search detection.
       # detection.
       #
+      # @return [Boolean]
+      #
       # @example
       #   require "gcloud"
       #
       #   gcloud = Gcloud.new
       #   vision = gcloud.vision
-      #   analysis = vision.detect image, safe_search: true
+      #   image = vision.image "./acceptance/data/face.jpg"
+      #
+      #   analysis = vision.annotate image, safe_search: true
       #   analysis.safe_search? #=> true
       #
       def safe_search?
         !safe_search.nil?
       end
 
-      # The Analysis::Properties results containing the results of properties
-      # detection.
+      ##
+      # The results of properties detection.
+      #
+      # @return [Properties]
       #
       # @example
       #   require "gcloud"
       #
       #   gcloud = Gcloud.new
       #   vision = gcloud.vision
-      #   analysis = vision.detect image, properties: true
+      #   image = vision.image "./acceptance/data/face.jpg"
+      #
+      #   analysis = vision.annotate image, properties: true
       #   properties = analysis.properties
       #
       def properties
         return nil unless @gapi["imagePropertiesAnnotation"]
         @properties ||= Properties.from_gapi(@gapi["imagePropertiesAnnotation"])
       end
 
-      # Whether there is a Analysis::Properties result for properties detection.
+      ##
+      # Whether there is a result for properties detection.
+      #
+      # @return [Boolean]
       #
       # @example
       #   require "gcloud"
       #
       #   gcloud = Gcloud.new
       #   vision = gcloud.vision
-      #   analysis = vision.detect image, properties: true
+      #   image = vision.image "./acceptance/data/face.jpg"
+      #
+      #   analysis = vision.annotate image, properties: true
       #   analysis.properties? #=> true
       #
       def properties?
         !properties.nil?
       end
 
+      ##
```

Comment:
Does it make sense to streamline the docs a bit by marking the `to_h` and `to_hash` methods as `@private`?